R/server.R

```diff
@@ -1,24 +1,3 @@
-#' Class representing a local web server route: path + callback.
-#' @keywords internal
-#' @rdname VitessceConfigServerCallbackRoute
-VitessceConfigServerCallbackRoute <- R6::R6Class("VitessceConfigServerCallbackRoute",
```

Comment:
Out of curiosity, what was this for?

Reply:
This was for in-memory web server responses (i.e. create the JSON, return it from the callback function, set as a GET route for the web server) but now all responses save to the file system first. I think this is helpful for debugging the file conversions and simplifying the export functions.

Reply:
Cool, thanks!